tempest/api/workloadmgr/restore/test_exclude_bootdisk_backup.py

```diff
@@ -0,0 +1,367 @@
+import json
+import os
+import sys
+import time
+import paramiko
+
+import yaml
+from oslo_log import log as logging
+
+from tempest import command_argument_string
+from tempest import config
+from tempest import reporting
+from tempest import tvaultconf
+from tempest.api.workloadmgr import base
+from tempest.lib import decorators
+from tempest.util import cli_parser
+from tempest.util import query_data
+
+sys.path.append(os.getcwd())
+
+LOG = logging.getLogger(__name__)
+CONF = config.CONF
+
+
+class WorkloadTest(base.BaseWorkloadmgrTest):
+
+    credentials = ['primary']
+
+    @classmethod
+    def setup_clients(cls):
+        super(WorkloadTest, cls).setup_clients()
+
+
+    def assign_floating_ips(self, vm_id, cleanup):
+        fip = self.get_floating_ips()
+        LOG.debug("\nAvailable floating ips are {}: \n".format(fip))
+        self.set_floating_ip(str(fip[0]), vm_id, floatingip_cleanup=cleanup)
+        return(fip[0])
+
+
+    def data_ops_for_bootdisk(self, flo_ip, data_dir_path, file_count):
+        ssh = self.SshRemoteMachineConnectionWithRSAKey(str(flo_ip))
+        self.install_qemu(ssh)
+        self.addCustomfilesOnLinuxVM(ssh, data_dir_path, file_count)
+        ssh.close()
+
+
+    @decorators.attr(type='workloadmgr_cli')
+    def test_exclude_bootdisk_backup(self):
```

Comment:
Rename the function name to exclude_bootdisk_backup() so that it does not get called during test suite execution.